ui/app/templates/allocations/allocation/task/index.hbs

```diff
@@ -105,6 +105,32 @@
     </div>
   {{/if}}
 
+  {{#if model.task.volumeMounts.length}}
+    <div data-test-volumes class="boxed-section">
+      <div class="boxed-section-head">
+        Volumes
+      </div>
+      <div class="boxed-section-body is-full-bleed">
+        {{#list-table source=model.task.volumeMounts as |t|}}
+          {{#t.head}}
+            <th>Name</th>
+            <th>Destination</th>
+            <th>Permissions</th>
+            <th>Client Source</th>
+          {{/t.head}}
+          {{#t.body as |row|}}
+            <tr data-test-volume>
+              <td data-test-volume-name>{{row.model.volume}}</td>
+              <td data-test-volume-destination><code>{{row.model.destination}}</code></td>
+              <td data-test-volume-permissions>{{if row.model.readOnly "Read" "Read/Write"}}</td>
```

Comment:
I know the CLI declares permissions via read_only = "true/false", but I'd be curious to see how CSI is phrasing permissions. However we're doing it, I think being consistent across host volumes and CSI, both in the CLI and the UI, is probably pretty important. I like "Read" and "Read/Write" better than the options I had in the original mocks FWIW!

Reply:
👍 CSI volumes _should_ follow suit, but I'll keep in eye out.